draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1104,6 +1104,8 @@ Outputs:
   : A base64 encoded `TransItem` of type `consistency_proof_v2`, whose
     `tree_size_1` MUST match the `first` input. If the `sth` output is omitted,
     then `tree_size_2` MUST match the `second` input.
+    If `first` and `second` are equaly, the returned consistency proof MUST
```

Comment:
Note that I think it should be OK to return an empty consistency path if the log implementation chooses to allow getting inclusion proofs between arbitrary tree sizes.